libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/AdaptiveDialog.cs

```diff
@@ -661,7 +661,7 @@ private async Task<bool> ProcessQueuesAsync(SequenceContext sequenceContext, Can
             }
             else
             {
-                evt = new DialogEvent() { Name = AdaptiveEvents.Ask, Bubble = false };
+                evt = new DialogEvent() { Name = AdaptiveEvents.EndOfActions, Bubble = false };
```

Comment:
>EndOfActions [](start = 64, length = 12)

EndOfPlan?

Reply:
Considered that, but thought we had moved away from the word plan.